app/templates/settings/password.hbs

```diff
@@ -1 +1 @@
-{{settings/password-section}}
\ No newline at end of file
+{{settings/password-section submit='changePassword' passwordCurrent=passwordCurrent passwordNew=passwordNew passwordRepeat=passwordRepeat isLoading=isLoading}}
```

Comment:
@niranjan94 When I use this, two requests are made to the server, out of which first one posts the body with given parameters while the second one sends empty body. Hence, I kept the previous action only. Please guide if I am missing something somewhere. Rest all I did. Please review.